adaptivefiltering/asprs.py

```diff
@@ -55,7 +55,12 @@ def asprs(*vals):
         A sorted tuple of integers with ASPRS codes:
     :rtype: tuple
     """
-    return tuple(sorted(set(sum((_asprs(v) for v in vals), ()))))
+
+    if is_iterable(vals):
```

Comment:
if you capture it as `*vals`, it is always a `tuple`. I think you meant to capture as `vals`.

Reply:
This also explains the CI error.